browser-test/src/header.test.ts

```diff
@@ -1,56 +1,59 @@
-import {test, expect} from '@playwright/test'
+import {test, expect} from './support/civiform_fixtures'
```

Comment:
This file extends the built-in Playwright test allowing the use of custom fixtures.

Instead of 

```typescript
test('Check screenshots and validate accessibility', async () => {
  const page = {ctx}
  // do stuff
}
```

We just tell the test what we want. Basically DI.

```typescript
test('Check screenshots and validate accessibility', async ({page}) => {
  // do stuff
}
```

Eventually everything will use this and not the default playwright one.

Reply:
Nice!

Reply:
I like how much cleaner this is.